docs/self-managed/operational-guides/production-guide.md

```diff
@@ -0,0 +1,530 @@
+
+- If you have a use case for enabling [Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) then it is recommended to do so.
+<!--Maybe link this to customer: https://github.com/ahmetb/kubernetes-network-policy-recipes-->
+- It is possible to have a pod security standard that is suitable to the security constraints you might have. This is possible through modifying the Pod Security Admission. Please refer to the [Kubernetes documentation](https://kubernetes.io/docs/concepts/security/pod-security-admission/) in order to do so.
+- By default, the Camunda Helm Chart, uses a read-only root file-system configuration for the pod. It is recommended to keep this default. No addition needs to be made in your `production-values.yaml`.
+- Disable privileged containers
+- It is possible to modify either the `containerSecurityContext` or the `podSecurityContext`. For example, here is a configuration for the core component that can be added to your `production-values.yaml`:
+
+```yaml
+podSecurityContext:
+  runAsNonRoot: true
+  fsGroup: 1001
+  seccompProfile:
+    type: RuntimeDefault
+
+containerSecurityContext:
+  allowPrivilegeEscalation: false
+  privileged: false
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 1001
+  seccompProfile:
+    type: RuntimeDefault
+```
+
+- It is best to only pull from a private registry such as [Amazon ECR](https://aws.amazon.com/ecr/) and not pull directly from Docker Hub. This is because it will increase the control you have over the images, will save you from rate limits, and increase performance and reliability. Furthermore, it is possible to restrict your cluster to only pull from known registries, this can be done using [Open Policy Agent](https://blog.openpolicyagent.org/securing-the-kubernetes-api-with-open-policy-agent-ce93af0552c3#3c6e), for example.
+- Open Policy Agent can also be used to have a [whitelist for ingress hostnames](https://www.openpolicyagent.org/docs/latest/kubernetes-tutorial/#4-define-a-policy-and-load-it-into-opa-via-kubernetes).
+
+## Observability and Monitoring
+
+Here are some points to keep in mind when considering observability:
+
+- It is possible to enable Prometheus on the Camunda Helm Chart. This can be added to your `production-values.yaml`:
+
+```yaml
+prometheusServiceMonitor:
+  enabled: false
+```
+
+- A tool such as [Loki](https://grafana.com/oss/loki/) could be used for the retention and archival of logs. It can also be used to aggregate logs.
+- It is possible to enable audit logging for Camunda components to track user actions, especially for sensitive data or configuration changes. Here is an example configuration to change the log level of optimize within the core components. This can be added in your `production-values.yaml`:
```

Comment:
Can we add an example to changs logging to all components? It is often asked for

Reply:
An example like that could get out of date very quickly. I think it is better to instead guide the customer to the respective application components to view how to modify the log levels. This will reduce repetition in our docs.
What do you think @theburi ?

Reply:
Ok. Let's leave it as it is. I think the Unified configuration project will address this in the future, and we don't have to worry about this.